test/xbgpu/test_engine.py

```diff
@@ -1153,13 +1224,24 @@ def heap_factory(batch_index: int, present: np.ndarray) -> list[spead2.send.Heap
             assert xbengine.sensors[f"{corrprod_output.name}.xeng-clip-cnt"].value == n_vis
 
     def _patch_get_in_item(
-        self, monkeypatch: pytest.MonkeyPatch, count: int, client: aiokatcp.Client, *request
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        count: int,
+        client: aiokatcp.Client,
+        requests: list[tuple[str, ...]],
     ) -> list[int]:
         """Patch :meth:`~.BPipeline._get_in_item` to make a request partway through the stream.
 
         The returned list will be populated with the value of the
         ``steady-state-timestamp`` sensor immediately after executing the
         request.
+
+        Parameters
+        ----------
+        count
+            Count point at which to execute katcp requests in `requests`.
```

Comment:
"Count point" isn't a clear term. How about
```suggestion
            Index (counting from 1) of the call to `_get_in_item` prior to which the `requests` will be made.
```